help/destinations/catalog/crm/salesforce.md

```diff
@@ -168,26 +173,40 @@ To correctly send your audience data from Adobe Experience Platform to the [!DNL
     * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on these attributes.
```

Comment:
[!UICONTROL Attribute name] should be in bold, like: `**[!UICONTROL Attribute name]**`

Also, please revise grammar and linking: 

`Refer to the [[!DNL Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on these attributes.`

Reply:
@vgiurgiu Updated verbiage, did not understand note about the linking, kindly elaborate.

Reply:
Sure. In the link here `[[!Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5)`, the DNL tag is missing, so we need to add it, as shown in the updated example below.

`Refer to the [[!DNL Salesforce CRM] documentation](https://help.salesforce.com/s/articleView?id=sf.custom_field_attributes.htm&type=5) for guidance on these attributes.`

Reply:
Thanks, duh of me !